app/src/main/res/values/styles.xml

```diff
@@ -646,6 +646,13 @@
         <item name="android:stateListAnimator">@null</item>
     </style>
 
+    <style name="TabTrayHeader16TextStyle" parent="TextAppearance.MaterialComponents.Body1">
```

Comment:
I am wondering couldn't we just say the parent is `Header16TextStyle` and change the textColor. Alternatively, just set the textColor where we use `TabTrayHeader16TextStyle` to override the styles and avoid creating a new style.